conf/jira.yaml.template

```diff
@@ -3,3 +3,6 @@ JIRA:
   URL: https://issues.redhat.com
   # Provide api_key to access Jira REST API
   API_KEY: replace-with-jira-api-key
+  COMMENT_TYPE: comment_type
```

Comment:
Can you add a yaml comment with the valid types?

Reply:
What is `Comment type` ?

Reply:
@jyejare The group or role to which this comment is visible.

Reply:
@jameerpathan111 Then wouldn't it be appropriate to rename to `COMMENT_VISIBILITY`?

Comment Type sounds really different than what it is made for.

Reply:
That's how Jira calls it and I'd suggest going with the same to avoid confusion.